foghorn-edge/scripts/bootstrap-edgemanager.sh

```diff
@@ -0,0 +1,12 @@
+yes n | cp -i /fh_vol/fh/tmp/certs/license-ca.crt /fh_vol/fh/certs/ca/ 
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.